datasets/msr_text_compression/README.md

```diff
@@ -0,0 +1,173 @@
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://msropendata.com/datasets/f8ce2ec9-7fbd-48f7-a8bb-2d2279373563
+- **Repository:**
+- **Paper:** https://www.microsoft.com/en-us/research/wp-content/uploads/2016/09/Sentence_Compression_final-1.pdf
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+
+This dataset contains sentences and short paragraphs with corresponding shorter (compressed) versions. There are up to five compressions for each input text, together with quality judgements of their meaning preservation and grammaticality. The dataset is derived using source texts from the Open American National Corpus (ww.anc.org) and crowd-sourcing.
+
+### Supported Tasks and Leaderboards
+
+Text Summarization
+
+### Languages
+
+English
+## Dataset Structure
+
+### Data Instances
+
+It contains approximately 6,000 source texts with multiple compressions (about 26,000 pairs of source and compressed texts), representing business letters, newswire, journals, and technical documents sampled from the Open American National Corpus (OANC1).
+
+- Each source text is accompanied by up to five crowd-sourced rewrites constrained to a preset
+compression ratio and annotated with quality judgments. Multiple rewrites permit study of the impact of operations on human compression quality and facilitate automatic evaluation.
+- This dataset is the first to provide compressions at the multi-sentence (two-sentence paragraph)
+level, which may present a stepping stone to whole document summarization.
+- Many of these two-sentence paragraphs are compressed both as paragraphs and separately sentence-bysentence, offering data that may yield insights
+into the impact of multi-sentence operations on human compression quality.
+
+| Description       | Source | Target | Average CPS | Meaning Quality | Grammar Quality |
+| :------------- | :----------: | -----------: | -----------: | -----------: | -----------: |
+|  1-Sentence | 3764   | 15523    | 4.12 | 2.78 | 2.81 |
+|  2-Sentence | 2405   | 10900    | 4.53 | 2.78 | 2.83 |
+
+**Note**: Average CPS = Average Compressions per Source Text
+
+### Data Fields
+```json
+{
+  "source_id": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "domain": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "source_text": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "targets": {
+    "feature_type": "Sequence",
+    "compressed_text": {
+      "feature_type": "Value",
+      "dtype": "string"
+    },
+    "judge_id": {
+      "feature_type": "Value",
+      "dtype": "string"
+    },
+    "num_ratings": {
+      "feature_type": "Value",
+      "dtype": "int64"
+    },
+    "ratings": {
+      "feature_type": "Sequence",
+      "feature": {
+        "feature_type": "Value",
+        "dtype": "int64"
+      }
+    }
+  }
+}
+```
```

Comment:
here we expect descriptions of the different fields in natural language

could you change that please ?

Reply:
fixed